api/krusty/fnplugin_test.go

```diff
@@ -82,6 +88,125 @@ spec:
 `)
 }
 
+func TestFnExecGeneratorWithOverlay(t *testing.T) {
+	fSys := filesys.MakeFsOnDisk()
+	th := kusttest_test.MakeHarness(t)
```

Comment:
```
th := MakeHarnessWithFs(t, fSys)
```

The use `th.WriteK`, etc.

see https://github.com/kubernetes-sigs/kustomize/blob/master/api/krusty/pluginenv_test.go#L27

Reply:
done